doc/ledger.txt

```diff
@@ -135,6 +135,13 @@ Tips and useful commands
   default commodity is set to `'$'`). Or you may press <Tab> in Visual mode
   to align a number of transactions at once.
 
+* `:LedgerAlignBuffer`
```

Comment:
Lets move this up in the help to be just below it's `:LedgerAlign` counterpart.

Also lets mention very briefly in the help blurb *why* this is different than just `:%call ledger#align_commodity()`, namely that it saves and restores the cursor position.

Reply:
Resolved in facc97f.